### PR TITLE
PLANET-6506: Remove Smartsheet code 

### PR DESCRIPTION
**Pull request by lithrel**
Ref: https://jira.greenpeace.org/browse/PLANET-6506

> Now that we switched to Google Sheets for the Program codes we should remove any Smartsheet code from our repos.
> Tasks
>    Remove any [Smartsheet code](https://github.com/search?q=org%3Agreenpeace+smartsheet&type=code) from theme, docker, builder, helm chart
>    Remove "Google Sheets" feature flag functionality. Google Sheets should work already by default.

Screenshot from 2022-07-11 14-07-32

I also added some exception capture to Sentry, to be aware of possible misconfigurations in prod.

## Test

- In Planet4 > Analytics, there's no option for Smartsheet ID
- In Planet4 > Features, there's no Google Sheet option
- Local projects are still available in the page editor

### CODE REVIEW

**Comment by planet-4** [comment]
### Test instance is ready :rocket:

:new_moon: [deimos](https://www-dev.greenpeace.org/test-deimos) | [admin](https://www-dev.greenpeace.org/test-deimos/wp-admin/) | [blocks report](https://www-dev.greenpeace.org/test-deimos/wp-admin/?page=plugin_blocks_report) | [CircleCI](https://app.circleci.com/pipelines/github/greenpeace/planet4-test-deimos) | [composer-local.json](https://github.com/greenpeace/planet4-test-deimos/blob/main/composer-local.json)

:watch: 2022.07.12 10:50:55

File: src/Migrations/M011RemoveSmartsheetOption.php [hunk 1]
```diff
@@ -18,8 +19,15 @@ class M011RemoveSmartsheetOption extends MigrationScript {
...
 	protected static function execute( MigrationRecord $record ): void {
+		// Toggle between GoogleSheet and SmartSheet.
 		$options = get_option( Features::OPTIONS_KEY );
 		unset( $options['google_sheet_replaces_smartsheet'] );
 		update_option( Features::OPTIONS_KEY, $options );
+
+		// Smartsheet ID and legacy toggle.
+		$options = get_option( Settings::KEY );
+		unset( $options['analytics_local_smartsheet_id'] );
+		unset( $options['google_sheet_replaces_smartsheet'] );
```
Comment (sagarsdeshmukh) [inline src/Migrations/M011RemoveSmartsheetOption.php:30]: Is `google_sheet_replaces_smartsheet` flag present in both Features::OPTIONS_KEY & Settings::KEY ?

**Comment by sagarsdeshmukh** [comment]
Hi @lithrel , Does it working for you on local?

> Local projects are still available in the page editor

The environment variables are missing on my current local dev env. setup 🤔 might be it's related to only my local setup, but just to confirm.

The PR looks good to me 👍

**Comment by lithrel** [comment]
@sagarsdeshmukh Ah right, to test locally you need to put in a `GOOGLE_SHEETS_KEY` constant a JSON key from a [service account](https://cloud.google.com/iam/docs/service-accounts) you have or create, and give it access to the spreadsheet you want to use - a bit tedious to configure, but if you have to fix a bug on it it's good to have that :)

**Review by sagarsdeshmukh — APPROVED** [review]
👍

File: src/Migrations/M011RemoveSmartsheetOption.php (hunk 1 above)
Comment (lithrel) [reply src/Migrations/M011RemoveSmartsheetOption.php:30]: Yes, I was surprised too when I saw the values there; but Features were once part of the planet4_options, and when we moved them out we still left the features values in options, we even still have [some code that does a fallback on it](https://github.com/greenpeace/planet4-master-theme/commit/5efca4b3538a2b52d64f2d87b85230221134a0d6#diff-ecf1d8c1cedf861e4a52c13d299a24fc41ebadbecc85d321eacbf40b4a3b3799R200-R203)

**Pull request merged**